Microsoft.HealthVault/Connection/IConnectionInternal.cs

```diff
@@ -10,6 +10,8 @@
 using System.Collections.Generic;
 using Microsoft.HealthVault.PlatformInformation;
 using Microsoft.HealthVault.Transport;
```

Comment:
System namespaces being first is a resharper setting and I think common

---
In reply to: [115855562](https://github.com/Microsoft/healthvault-dotnetstandard-sdk/pull/27#discussion_r115855562) [](ancestors = 115855562)